index.html

```diff
@@ -670,9 +684,10 @@ <h2>Profiles</h2>
         </p>
 
         <pre class="example nohighlight" title="Basic structure of a profile">
-{
-  "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
-  "type": ["VerifiableProfile"],
+{"verifiableProfile": {
+  "id": "< profile id >",
+  "holder": "< holder id >",
```

Comment:
Why is the holder having to identify themselves here?

Reply:
I think this is essential. We already know who the subject is, from the claim. But we do not know who the holder is unless the data model tells us this. If the IDs are the same then the verifier knows that the holder is asserting that subject EQ holder. If the IDs are different, then either the subject has multiple IDs, or subject NE holder. Either way it is then up to the verifier to sort out the problem. But at least the holder and data model is giving it some guidance. There have been other suggestions to insert a relationship property here, so that when the IDs are different, the holder can assert what the relationship is. I think this is a good idea as it further helps the verifier to determine who the holder is.